wimm.Guardian.UnitTests/StringExtensionTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace wimm.Guardian.UnitTests
+{
+    [TestClass]
+    public class StringExtensionTests
+    {
+        private const string _name = "paramName";
+
+        [TestMethod]
+        public void IsNotNullOrWhitespace_TargetNull_Throws()
```

Comment:
In keeping with your pattern below, should this be called `IsNotNullOrWhitespace_IsNull_Throws`?